http/vibe/http/common.d

```diff
@@ -336,13 +337,222 @@ class HTTPStatusException : Exception {
 	string debugMessage;
 }
 
+final class MultiPartBodyPart
```

Comment:
I definitely wouldn't mind that. If someone should ever actually need them, they can still be added.